src/main/java/com/tysng/duke/ui/Response.java

```diff
@@ -72,6 +114,11 @@ public static Response newListing(List<Task> items) {
         return new Response(content);
     }
 
+    /**
+     * Generates a Echo type response.
+     *
+     * @return a response
```

Comment:
I like how you have a Response class for hardcoding static output. I personally only had a UI class in which I hardcoded private static final Strings as the output and called it within the class itself.